backend/src/jobs/start_recommitment.ts

```diff
@@ -55,7 +55,13 @@ import { RecommitmentService } from '../recommitment/recommitment.service';
     } else {
       logger.log('Starting recommitment job', 'Start Recommitment');
 
-      const data = await recommitmentService.handleStartRecommitment();
+      const ministry = process.env.RECOMMITMENT_MINISTRY || undefined;
```

Comment:
if we're passing ministry in the test run we'll need to move this up and pass it in